applications/CoSimulationApplication/python_scripts/base_classes/co_simulation_convergence_accelerator.py

```diff
@@ -0,0 +1,67 @@
+from __future__ import print_function, absolute_import, division  # makes these scripts backward compatible with python 2.6 and 2.7
+
+# Importing Kratos
+import KratosMultiphysics as KM
+
+# CoSimulation imports
+import KratosMultiphysics.CoSimulationApplication.co_simulation_tools as cs_tools
+import KratosMultiphysics.CoSimulationApplication.colors as colors
+
+class CoSimulationConvergenceAccelerator(object):
```

Comment:
I think that it would make sense that the convergence accelerators in the CoSimApp API matches the one already existent in the Kratos convergence accelerator base class `convergence_accelerator.hpp`.

Reply:
done in #5187